### ISSUE TITLE
Query Failure

### ISSUE THREAD

**Issue by karafecho** [RobokopU24/Feedback#108]
This issue is to report a ROBOKOP query that I ran multiple times but that never returned answers (even "no answers are available"). Note that the image below was captured after the query was submitted to ROBOKOP and after ROBOKOP attempted to fetch answers. Note also that Nyssa was able to run this query through ExEmplar, even with more specific predicates.

PUBCHEM.COMPOUND:6618 = Tetrabromobisphenol A - related_to - NamedThing - related_to - Gene - related_to - MONDO:0004995 = cardiovascular disease

![image](https://user-images.githubusercontent.com/26254388/216658065-d5826223-1a28-46c3-a16c-441644d7afe7.png)

**Comment by cbizon** [comment]
Query 1 is very big.  If I set the NamedThing to Gene, then there are 18000 results in ROBOKOP.

With NamedThing it is worse for multiple reasons, including some subclass-of edges that we probably should do something about.  Exemplar limits to 100 results (arbitrarily) so that is not a good comparison.

**Comment by cbizon** [comment]
I'm not sure why query 2 is dragging, seems like it should be good...

**Comment by cbizon** [comment]
This one actually does return

<img width="1545" alt="image" src="https://user-images.githubusercontent.com/10744350/225424605-eb3c19a1-758c-4ccb-a025-8a7a85e73379.png">

It's maybe not as quick as I'd like, but it is returning lots of results.   There do seem to be duplicates that I would hope would be filtered out...

**Comment by karafecho** [comment]
I didn't make the connection with the limit that Nyssa (or ExEmplar by default) placed on the number of returned results. Thanks for pointing that out. However, if ROBOKOP can place a default limit on the number of results it returns, based on something smart like scores, and then notifies the user that they can ask for additional results or wait for them, kind of like the initial version of ROBOKOP did and the Translator UI currently does, then that might be helpful for users. The email notification that the initial version of ROBOKOP did of "we will send you an email when your results are ready" would also work, but that would require a login option, I think.

I believe there are two other related issues here. 

The first one is strictly reliable performance. For instance, the query above ran for me the first time I ran it, but it then stopped running the second time I attempted to run it. 

The second issue is related to the first issue. Presumably, ROBOKOP timed out the second time I ran the query, but it did not give the user any indication as to why the query suddenly stopped running, e.g., no error message was returned, no "come back later" message was returned, nothing.
